lib/python/mod_python/psp.py

```diff
@@ -271,7 +271,7 @@ def __init__(self, label, file, cache):
                     psp.error_page.run({"exception": (et, ev, etb)}, flush)
                 else:
                     if PY2:
-                        raise et, ev, etb
+                        exec('raise et, ev, etb')
```

Comment:
i don't think so, syntax `raise et, ev, etb` is Python2 only. 
http://python-future.org/compatible_idioms.html#raising-exceptions

so, when using psp.py in Python3 (e.g. `import psp`), Python3 interpreter raises a SyntaxError.